src/dropdown/dropdown.ts

```diff
@@ -1,12 +1,12 @@
 import {Directive, Input, HostListener} from 'angular2/core';
 
-@Directive({selector: '[ngb-dropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
+@Directive({selector: '[ngbDropdown]', exportAs: 'ngbDropdown', host: {'[class.open]': 'open'}})
```

Comment:
Do I want? No, I don't want any of this, this is all BS. Do we need to do it? Yes.